src/Bicep.Core/Emit/EmitResult.cs

```diff
@@ -11,4 +13,5 @@ namespace Bicep.Core.Emit;
 public record EmitResult(
     EmitStatus Status,
     ImmutableArray<IDiagnostic> Diagnostics,
+    IFeatureProvider? Features,
```

Comment:
Is this needed?

Reply:
I'm using it to validate output in tests. It could probably be moved off this model but because the emitter consumes a semantic model and the diagnostics are tied to the model, I figured including a reference to features here was OK. Previously there were hardcoded expectations that now depend on a feature flag.